build/vanagon/components/rubygem-activemodel.rb

```diff
@@ -0,0 +1,4 @@
+component 'rubygem-activemodel' do |pkg, settings, platform|
```

Comment:
stupid question for all those gems, but:
* Do we vendor the latest version in every case?
* Is there some magic to check if an update is published on rubygems / how do we ensure to not ship outdated gems?

Reply:
Can't vendor the latest version in every case b/c of dependencies. This will have to be manually maintained for the time being. I plan on doing an update to the Gemfile, Gemfile.lock, and these files after the initial package is built.

Reply:
I should add that utilizing this format to install the gems is required to make sure the gems are built against the correct C libraries. Then the installer does a `bundle install` as part of the post-install scripts

Reply:
might be worth an entry in the README.md or a seperate HOW_DO_I_PACKAGE_THIS.md ?

Reply:
Yeah, that will come a bit "later". This is just the initial push and package deb to make sure everything works without a manual change. Then create package files for other "platforms" + the README. Vanagon desperately needs it